### DISCUSSION TITLE
[v8] Row selection outside useReactTable scope

### DISCUSSION THREAD

**Discussion by AmitNeuhaus** [TanStack/table#4286] [Q&A]
Hello,
I saw in the examples the ability to use a custom state and override `onRowSelectionChange`.
However this accepts only an updater function which results the custom state to contain only the ids of the selected rows.
How can i access the row values themself? (without returning to the table scope to access table.getSelectedFlatRows() ,etc..

Thank you!

**Comment by livingmine** [comment]
@AmitNeuhaus what do you want to achieve?
